common.gypi

```diff
@@ -1,6 +1,6 @@
 {
   'variables': {
-    'asan%': 0,
+    'asan%': 'false',
```

Comment:
You should keep this an int because that's what V8 expects (it inherits it.)

You can assign it in configure with `o['variables']['asan'] = int(options.enable_asan)`.